packages/StackedBar/src/stacked-bar.scss

```diff
@@ -20,14 +33,43 @@
   --ids-stacked-bar-no-color-hover: #{tokens.$grey-400};
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-stacked-bar-font-family: var(--hop-body-xs-semibold-font-family);
+  --ids-stacked-bar-font-size: var(--hop-body-xs-semibold-font-size);
+  --ids-stacked-bar-font-weight: var(--hop-body-xs-semibold-font-weight);
+  --ids-stacked-bar-color: var(--hop-neutral-text);
+  --ids-stacked-bar-color-ends: #FFFFFF; /* Missing semantic token */
+  --ids-stacked-bar-height: 1.5rem;
+  --ids-stacked-bar-height-small: 0.5rem;
+  --ids-stacked-bar-empty-fill-side: var(--hop-neutral-surface-disabled);
+  --ids-stacked-bar-empty-fill-middle: var(--hop-neutral-surface-weak);
+  
+  /* Bar Colors */
+  --ids-stacked-bar-strength-minus-2: #FF8E8E; /* Missing semantic token */
```

Comment:
There should be decorative tokens.

Reply:
I started using decorative tokens for some, but in reality, we need to add data vix tokens https://www.figma.com/file/he7mOkVFtqk0m3dgWKLcXk/%F0%9F%90%B8-Data-visualization?type=design&node-id=40%3A1101&mode=design&t=aWERRvcA3IcZ0HRa-1

Reply:
They should be in Hopper tokens I think

Reply:
yes eventually we need to add them. I'm not sure who or when, but I wasn't going to delay this PR for that. That's why I put the comments, so we can easily track it down later.

Reply:
Perfect, let's just make sure it's tracked.